smoke-tests-otel-starter/spring-boot-3/src/test/java/io/opentelemetry/smoketest/GraalVmNativeMongodbSpringStarterSmokeTest.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.smoketest;
+
+import org.junit.jupiter.api.condition.EnabledInNativeImage;
+
+/**
+ * GraalVM native image doesn't support Testcontainers in our case, so the docker container is
+ * started manually before running the tests.
+ *
+ * <p>In other cases, it does work, e.g. <a
+ * href="https://info.michael-simons.eu/2023/10/25/run-your-integration-tests-against-testcontainers-with-graalvm-native-image/">here</a>,
+ * it's not yet clear why it doesn't work in our case.
+ *
+ * <p>In CI, this is done in reusable-native-tests.yml. If you want to run the tests locally, you
+ * need to start the container manually: docker run -d -p 27017:27017 --name mongo --rm mongo:latest
+ */
+@EnabledInNativeImage
+public class GraalVmNativeMongodbSpringStarterSmokeTest
```

Comment:
As discussed, in this case, the test method should only be evaluated if an environment variable is set. The environment variable should be set in the Github action. 

The reason is  not to have a failing test if we execute this test locally with GraalVM native and without having started the MongoDB container.

Reply:
good catch!